### PR TITLE
Use release tool for updating json, index and latest on Windows

### PR DESCRIPTION
**Pull request by zanderz**
Look, Windows is on the latest list now:
https://s3.amazonaws.com/prerelease.keybase.io/index.html

Getting the right tools together to use the .sh scripts on Windows didn't seem worth it, since most of the interesting stuff is already in Go anyway.

@gabriel @maxtaco

### CODE REVIEW

File: packaging/windows/doinstaller_gui.cmd [hunk 1]
```diff
@@ -44,5 +45,10 @@ IF %ERRORLEVEL% NEQ 0 (
 )
 "%ProgramFiles(x86)%\Inno Setup 5\iscc.exe" /DMyExePathName=%PathName% /DMyAppVersion=%BUILDVER% /DMySemVersion=%SEMVER% /DNewDokanVersion=%DOKANVER% "/sSignCommand=signtool.exe sign /tr http://timestamp.digicert.com $f" %GOPATH%\src\github.com\keybase\client\packaging\windows\setup_windows_gui.iss
 
-:: Afterwards, do:
-:: winresource.exe -u [installer.exe] > update-windows-prod.json
+:: After sanity checking, do:
+
+::%GOPATH%\bin\windows_386\release update-json --version=%KEYBASE_VERSION% --src=Output/keybase_setup_gui-1.0.13-20160218110300+95d6179.386.exe --uri=https://s3.amazonaws.com/prerelease.keybase.io/windows-updates > update-windows-prod.json
```
Comment (zanderz) [inline packaging/windows/doinstaller_gui.cmd]: Still yet to automate this part even more, obvs.

File: go/winresource/UpdateJson.go [hunk 2]
```diff
@@ -1,71 +0,0 @@
-// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```
Comment (maxtaco) [inline go/winresource/UpdateJson.go]: why did this go bye-bye?

Comment (zanderz) [inline go/winresource/UpdateJson.go]: I hadn't realized that github.com/keybase/release does this already.

**Comment by maxtaco** [comment]
Ok, good to go then

**Pull request merged**